scripts/tests/service_tests.py

```diff
@@ -885,6 +885,22 @@ def test_kibana_elasticsearch_urls(self):
         self.assertTrue("elasticsearch" in kibana['depends_on'])
         self.assertEqual("elasticsearch01:9200,elasticsearch02:9200", kibana['environment']["ELASTICSEARCH_URL"])
 
+    def test_kibana_port(self):
+        kibana = Kibana(version="7.3.0", release=True, kibana_port="1234").render()["kibana"]
```

Comment:
Is it possible to extract version to a variable? That will allow changing one place in future modifications

Thinking out loud, even the kibana object could be extracted to a global variable

Reply:
in most cases, it is a value that has to match with the result of the test, if we move it to a class field we will still have to modify every assert to match the new version, or we make the result string in runtime.

Reply:
BTW the version is not important for the test, I mean it could be any value does not change anything (e.g. `10.100.1` is also valid).